source/common/http/conn_manager_impl.cc

```diff
@@ -2283,7 +2295,7 @@ bool ConnectionManagerImpl::ActiveStreamFilterBase::commonHandleAfterHeadersCall
     // Set headers_only to true so we know to end early if necessary,
     // but continue filter iteration so we actually write the headers/run the cleanup code.
     headers_only = true;
-    ENVOY_STREAM_LOG(debug, "converting to headers only", parent_);
+    ENVOY_STREAM_LOG(debug, "converting to headers only", parent_.parent_);
```

Comment:
This is one of those places where  I question the envoy "naming things parent" paradigm

Especially if we think this is going to be some interface class later, do you think it's worth having more descriptive naming?

Reply:
I think my idea was to introduce a new interface class to handle calling back out from the FM, and then phase the usages of `parent_` over to that. I could change it to `active_stream_` or something like that for now, wdyt?

Reply:
Yeah agreed might be worth it to use a more descriptive name. Otherwise in general this change makes sense to me. Thanks for taking this on!